test_n13_api.py

```diff
@@ -26,3 +24,6 @@ def test_user_agent(config):
     data = r.json()
     assert data == {u'user-agent': 'python-requests/' + str(requests.__version__)}
     assert r.elapsed.total_seconds() < 0.500
+
+
+
```

Comment:
whoa! you cleanup above and introduce empty lines here... :(

Reply:
i will work on adding static code analysis in place to prevent merge requests if code is not per PEP8 with minor exceptions.